src/QueryReflection/BasePdoQueryReflector.php

```diff
@@ -136,6 +136,15 @@ public function getResultType(string $queryString, int $fetchType): ?Type
         return $arrayBuilder->getArray();
     }
 
+    public function setupDbaApi(?DbaApi $dbaApi): void
```

Comment:
I think we should leave the method in the base class but implement it in the subclasses, so we don't need this ugly if-condition which couples the base class to the actual subclasses